package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "mod-sandbox",
+  "version": "0.0.0",
+  "description": "Small description for mod-sandbox goes here",
+  "private": true,
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build",
+    "start": "ember server",
+    "test": "ember test"
+  },
+  "repository": "",
+  "engines": {
+    "node": ">= 0.10.0"
```

Comment:
I would make this something more along the lines of `>= 4` or maybe even `>= 6` to keep with the times.